apps/events/tables.py

```diff
@@ -43,10 +44,16 @@ def render(self, value, record):
         )
 
 
+class ParamsColumn(tables.Column):
+    def render(self, value, record):
+        items = [f"<li><strong>{k}</strong>: {v}</li>" for k, v in value.items()]
+        return format_html(f"<ul>{''.join(items)}</ul>")
```

Comment:
`items` is the one that needs format_html. Also you can use [format_html_join](https://docs.djangoproject.com/en/4.2/ref/utils/#django.utils.html.format_html_join):

```suggestion
        items = format_html_join("", "<li><strong>{}</strong>: {}</li>", value.items())
        return f"<ul>{items}</ul>"
```

Reply:
I still need to use `format_html` on the `<ul>` here, otherwise it looks like this: 
![image](https://github.com/dimagi/open-chat-studio/assets/146896/4bf4afab-536e-476c-ac65-84827415fa9e)
Thanks for the tip about `format_html_join` - I didn't know about that one! 

https://github.com/dimagi/open-chat-studio/pull/468/commits/a5f56ec112ee19bffce7addfad2cbeb6cba65e48

Reply:
> I still need to use format_html (to mark the string as safe) on the <ul> here

Oh right, you could just use `mark_safe` for that since it's not actually doing any formatting.